### PR TITLE
chore: migrate passes to template

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: templates/mobile/src/passes/AddToAppleWalletButton/index.js [hunk 1]
```diff
@@ -10,8 +10,7 @@ import { withApollo } from '@apollo/client/react/hoc';
...
...QlDIqzieYBTKazKNFhnRzjSEY2STKu1ZnH7XM/IKUWtVMCuBAYp803u57AsdwpoEQvY3XStmx9EC/lxkLW6gIdSMYyG3yoQFAn/bQ1g482DoSsDXlsyn9/rMQZ1+fYt3JYjRb+ApqWrA2B2YIKMwcV/UxlIKHIoTFNHozgEdSiKOmkUYK9cvJgqKfm9mDouAM5Uye1rKERf9+2Fa/2fLdy8tieb8Hd344s+bMeRoU8un49jcNcw7YtGaynoud5ac6I3ReSS/G1pBNpFKTZSMpVlHQIVJFHFXWeOI+z9yg9HU3T9N/u8/lin+M6aC3QTK2hItNrKD8PvVrlGNbbsiIZxSgG4hwV9HyOH+a1ZS6RyyGfz0PSSzFV6wzbgomxPxwFzzCoQEtbr6xZvHqju9OiAcIqQc7Owvzxe5jnJiEJLCogw3i0ubQ59AbuNTVD7N0Ht6sbXjJFFZiuVMODclIpurdArljsL4Md92zb0uPVaVY/2pQvIv3xh7Qq8K4Ix1AMxDPEckn/oYdR7O8nb117Gl4oYGdQoGvz+wp8ucYM8k8/Q0+fysYKOiwOPFUioKJDM+SRx1Do610TKL5+Q0ODq8FoWq5mB+tGbkVTonTwcQ3H00zHDvXcgjF7ir1UdBxgcDeK3Z2Rrx9FMZ1OlwKPRdGPrLOwdQvsPUN6mjEEYzEkbxxLlutqKDfbDef+ByJftxbF0JtHLSdeSXfhrkE0tW6G89UXOgewfpAobI6pwXuB/fvhUjZej6YTxpnx445lWcZ63CAhTSTPTkKdO0crBguqfRu87dthhT/crumtE+lNN62bxwJLbc+B3dEO8FZpaxbSlStePliXGLv8Jtdj/8YG489YN1JraXH9OkZQ117uY/RkbLvg1zHDkPH+MlrjQ2tv3zWn56CURvmrfY1XiKG6mUgoWkJZGkwY4pMY2nhVJqVSGb3012DZbP/LyVS67uOM/kmhaPa9wU/E9xi5Lt2QeZYWaetfOa/KD9FOamps/j3b1XeEtUNrsKmp8XcLC/nnbLsUkke77PXTophSjQ0tJ7M9fTt5/B8wFly6dH5XMTf/tWUtdq7F4nM9cQ3DVKlUOpdIJt/qyPa+tvxeq3pmcvLMnQbkreTXrJRuea2+/PT/e+wpWvHQ50RvhoJpQqnUT9lstlDNin8Bjwadt7VBWwgAAAAASUVORK5CYII=',
```
Comment (calebpanza) [inline templates/mobile/src/passes/AddToAppleWalletButton/index.js:13]: First time seeing this code so I'm just curious. Why is this data string hardcoded?

**Review by calebpanza — COMMENTED** [review]
I think this is good! Is there a way to test it?

File: templates/mobile/src/passes/AddToAppleWalletButton/index.js (hunk 1 above)
Comment (redreceipt) [reply templates/mobile/src/passes/AddToAppleWalletButton/index.js:13]: not sure

**Comment by redreceipt** [comment]
> I think this is good! Is there a way to test it?

yea...I didn't test it. I just moved it over. not sure if it's worked for a while. but the app boots after the migration which validates imports and links are all still functioning which is the purpose of this PR

**Review by tristanguckenberger — APPROVED** [review]
Tested on my local, looks good to me!

**Pull request merged**